### ISSUE TITLE
Axe Linter throwing ARIA `role="text"` violation warning even when valid

### ISSUE THREAD

**Issue by amoore108** [dequelabs/axe-core#3200]
Product: **Axe Linter**

Expectation: Axe Linter should not report `aria-roles` violations when using `role="text"` correctly.

Actual: Axe Linter is reporting a `Axe Linter (aria-roles): Ensures all elements with a role attribute use a valid value` violation even though the semantic structure is correct:
<img width="1108" alt="Screenshot 2021-10-12 at 15 21 32" src="https://user-images.githubusercontent.com/11633031/136974128-52eb4b07-245e-4bec-a847-3b95d3d27e11.png">

Motivation: PR https://github.com/dequelabs/axe-core/pull/2702 added functionality to allow usage of the `role="text"` rule. This doesn't appear to be reflected in the Axe Linter VS Code extension, and is throwing errors when it shouldn't.

<pre>
axe-core version: 4.3.3
axe linter version: 4.2.1
